src/tool_operate.c

```diff
@@ -1265,7 +1494,21 @@ static CURLcode single_transfer(struct GlobalConfig *global,
         if(result)
           break;
 
-        use_proto = url_proto(per->this_url);
+
+        /* it might have a result from a previous iteration, free proto */
+        if(use_proto.proto) {
+          Curl_safefree(use_proto.proto);
+          use_proto.result = CURLE_OK;
+        }
+
+        use_proto = url_proto(&per->this_url, config);
+
+        if((!strcmp(use_proto.proto, "ipfs")
+             || !strcmp(use_proto.proto, "ipns"))
+            && use_proto.result) {
```

Comment:
```suggestion
        if((!strcmp(use_proto.proto, "ipfs") ||
            !strcmp(use_proto.proto, "ipns")) &&
           use_proto.result) {
```

We usually put the operands at the end of the line and indent the following line 
according to the open paren in the previous line.